WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift

```diff
@@ -579,3 +579,11 @@ private extension String {
         static let title = NSLocalizedString("Loading Plugin...", comment: "Text displayed while loading an specific plugin")
     }
 }
+
+extension PluginViewModel {
+    func reloadPlugin() {
+        state = .loading
+        let store = StoreContainer.shared.plugin
```

Comment:
We are already receiving an instance of the store on init. It would be good to use that one.
One idea would be to capture it on a closure and then call the closure, or just to store it on a property.